wire/msgmixpairreq.go

```diff
@@ -83,7 +85,8 @@ func (msg *MsgMixPairReq) Pairing() ([]byte, error) {
 		VarIntSerializeSize(uint64(len(msg.ScriptClass))) + // Script class
 		len(msg.ScriptClass) +
 		2 + // Tx version
-		4 // Locktime
+		4 + // Locktime
+		1 // Pairing flags
```

Comment:
What about flags?

Reply:
the non-pairing flags are intentionally not a part of the pairing.

the pairing is a compatibility advertisement so that only peers that will create compatible coinjoins with compatible mixing protocols will work together.  the non-pairing flag in this case is being used to advertise support for solving the slot reservation polynomial, which doesn't affect the compatibility, but at least one peer in a session is required to support solving the poly and publishing the results.

Reply:
Make sense.  Thanks for the explanation.